src/axom/quest/InOutOctree.hpp

```diff
@@ -1608,10 +1647,11 @@ void InOutOctree<DIM>::insertMeshTriangles()
               QUEST_OCTREE_DEBUG_LOG_IF(
                 DEBUG_BLOCK_1 == childBlk[j] || DEBUG_BLOCK_2 == childBlk[j],
                 //&& tIdx == DEBUG_TRI_IDX
-                "Added triangle " << tIdx << " @ " << spaceTri << " with verts "
-                                  << m_meshWrapper.triangleVertexIndices(tIdx)
-                                  << "\n\tinto block " << childBlk[j]
-                                  << "\n\twith data " << *(childDataPtr[j]));
+                fmt::format("Added triangle {} @ {} ", tIdx, spaceTri)
+                  << " with verts " << m_meshWrapper.triangleVertexIndices(tIdx)
```

Comment:
Is there a reason you nested these fmt calls?

Reply:
Thanks for noticing it.

`fmt` didn't know how to format this. It is a slam array with an ostream operator. I think I can get this working by adding `fmt/ostream.h`.

I'll look into it.

Reply:
Adding `fmt/ostream.h` worked -- and this looks a lot better now.  Thanks!